charter.md

```diff
@@ -7,7 +7,7 @@ Steering Committee.
 
 The Kubernetes Steering Committee is the governing body of the Kubernetes
 project, providing decision-making and oversight pertaining to the Kubernetes
-project bylaws, sub-organizations, and financial planning.  The Steering 
+project by laws, sub-organizations, and financial planning.  The Steering 
```

Comment:
"bylaws" is correct

Reply:
oh. I always read it as by-laws. Anyways, will revert it. thanks :)